lib/originals-helper.js

```diff
@@ -56,6 +56,75 @@ function getModuleNameForType(type) {
   }
 }
 
+/**
+ * @param {Node} node - a ts-morph AST node
+ * @param {string} property - the property to find on some module
+ *
+ * @return {string} the get-originals module name corresponding to the node's
+ * type. (This is sometimes different from the TypeScript type name.)
+ */
+function getModuleNameForNodeFromPrototypeChain(node, property) {
+  return getModuleNameForTypeFromPrototypeChain(node.getType(), property);
+}
+
+/**
+ * @param {Type} type - a ts-morph type
+ * @param {string} property - the property to find on some module
+ *
+ * @return {string} whether the property exists on the given type
+ */
+function checkPropertyOnType(type, property) {
+  for (const symbol of type.getProperties()) {
+    if (property === symbol.getName()) {
+      return true;
+    }
+  }
+  return false;
+}
+
+/**
+ * @param {Type} type - a ts-morph type
+ * @param {string} property - the property to find on some module
+ *
+ * @return {string} the get-originals module name corresponding to the
+ * type. (This is sometimes different from the TypeScript type name.)
+ */
+function getModuleNameForTypeFromPrototypeChain(type, property) {
```

Comment:
I don't think we need to factor out a separate Type function instead of a Node function. I did that for the other one because we needed to use both at different places in the code. But you only export the Node version, so probably we should just have one.